### ISSUE TITLE
Its possible to play aaxc files without aax2mp3?

### ISSUE THREAD

**Issue by tdguchi** [mkb79/audible-cli#125]
Not exactly an issue maybe a question...

the aaxc files, could be played as is? something like aax2mp3 but "on the go" 

thanks.

**Comment by mkb79** [comment]
aax and aaxc files are encrypted containers which contains the audio, metadata and cover. They have to be decrypted before they can be played. This can be done with aax2mp3, ffmpeg or other tools. Maybe you can try to pipe the decrypted ffmpeg output to a software player.

**Comment by tdguchi** [comment]
yeah, i made it, using key/iv pair, now i only need to  extract every pair when playing
